### PR TITLE
py-earth2mip: add new package

### PR DESCRIPTION
**Pull request by adamjstewart**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-nvidia-dali/package.py [hunk 1]
```diff
@@ -16,6 +16,8 @@ class PyNvidiaDali(PythonPackage):
     homepage = "https://developer.nvidia.com/dali"
     url = "https://developer.download.nvidia.com/compute/redist/"
 
+    skip_version_audit = ["platform=darwin"]
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-nvidia-dali/package.py:19]: I wasn't aware of this feature.  Nice.

**Review by tldahlgren — DISMISSED** [review]
Confirmed new homepages and version sha256s/commits. Rest LGTM.

**Comment by tldahlgren** [comment]
@jcpunk @skosukhin Do you want to comment on this PR before it is merged?

File: var/spack/repos/builtin/packages/py-nvidia-dali/package.py (hunk 1 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-nvidia-dali/package.py:19]: @alalazo added it for CUDA

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/py-nvidia-dali/package.py:19]: Then I probably heard about it during a meeting but this is the first time I've seen it in use.

**Comment by adamjstewart** [comment]
Down to only 2 remaining build issues: onnxruntime and tensorstore

**Comment by adamjstewart** [comment]
Okay, everything builds now.

**Comment by jcpunk** [comment]
No objection from me.

**Review by tldahlgren — APPROVED** [review]
Re-confirmed homepages and version directives.

**Pull request merged**